thirdparty/libjpeg-turbo/src/jconfigint.h

```diff
@@ -33,10 +33,18 @@
 #define PACKAGE_NAME  "libjpeg-turbo"
 
 /* Version number of package */
-#define VERSION  "3.1.1"
+#define VERSION  "3.1.0"
```

Comment:
Looks like I missed this in the original libjpeg-turbo PR after going back to the stable version.